addons/mail/static/src/xml/composer.xml

```diff
@@ -15,7 +15,7 @@
                     <h4>Drag Files Here <span class="fa fa-download"/></h4>
                 </div>
                 <div class="o_composer_input">
-                    <textarea class="o_input o_composer_text_field" tabindex="2" placeholder="Write something..."/>
+                    <textarea class="o_input o_composer_text_field" tabindex="2" t-att-placeholder="widget.options.isLog ? 'Log an internal note...' : 'Send a message to followers...'"/>
```

Comment:
Those strings need to be translatable. It was automatically done before when directly set on a `placeholder` attribute, but this does not work automatically when using JS strings.